src/patternfly/components/Content/styles.scss

```diff
@@ -1,36 +1,168 @@
 @import "../../patternfly-utilities";
 
+// stylelint-disable
+:root {
+
+  // line height - DELETE - just here for review with designer
+  --pf-c-content--sm--LineHeight: var(--pf-global--sm--LineHeight);
+  --pf-c-content--md--LineHeight: var(--pf-global--md--LineHeight);
+  --pf-c-content--lg--LineHeight: var(--pf-global--lg--LineHeight);
+
+// are we being opinionated by specifying a single font-weight variable for all headers?
+  // Headings
+  --pf-c-content--heading--FontWeight: var(--pf-global--normal--FontWeight);
+
+  // Hero title
+  // --pf-c-content--hero-title--MarginTop: var(--pf-global--spacer--xxxl);
+  // --pf-c-content--hero-title--MarginBottom: var(--pf-global--spacer--lg);
+  // update variables above when spacers are updated
+  // the following are spacers as defined in the issue
+  --pf-c-content--hero-title--MarginTop: 24px;
+  --pf-c-content--hero-title--MarginBottom: 16px;
+  --pf-c-content--hero-title--LineHeight: var(--pf-global--sm--LineHeight);
+  --pf-c-content--hero-title--FontSize: var(--pf-global--hero-title--FontSize);
+  --pf-c-content--hero-title--FontWeight: var(--pf-global--normal--FontWeight);
+
+  // h1 - Main title
```

Comment:
In the context of the Contents component, where we use elements instead of classes as selectors, it seems like we should use the element names in the variable names. 

@andresgalante do you have any thoughts about that?

Reply:
You are right. The variable reflects the selector, the idea been that you can read the html API and undertand which variable  to change.

Just make sure that the component variable with the element selector is set with a global variable.